src/proc/mod.rs

```diff
@@ -8,6 +8,7 @@ mod terminator;
 mod typifier;
 
 pub use index::IndexableLength;
+pub use interpolator::apply_default_interpolation;
```

Comment:
this logic could also live in `src/front/mod.rs` (or `src/front/interpolator.rs`), which is more specific, right?

Reply:
Sure. Moved to `src/front/interpolator.rs`.